### DISCUSSION TITLE
New commands like /stonecutter, /loom etc.

### DISCUSSION THREAD

**Discussion by bobhenl** [Zrips/CMI#3980] [Feature Requests]
**Description of feature/improvement request:**

---
### Detailed explanation of what it should do, when and in what situation: ###
Hi, not sure if it's possible, but would be nice if you add something like virtual beacon. We already have merchant, anvil, enderchest, workbench, what about next virtual things, for ex. Beacon, stonecutter etc.? Should work like normal items, so they will open you guy, beacon have to be somehow more improved to work, but for ex. stonecutter, barrel (as backpack), blast furnace, cartography table, grindstone, loom, smoker, smithing table, furance, brewing stand etc.?

### Basic explanation as to why this feature is needed and/or useful to have: ###
It's needed like /ec, /workbench etc., It's nice like VIP perk to allow players open these interactive containers without having them built on the ground. For ex. if players will want disenchant something, they will use /stonecutter, etc.

**Comment by CloudeLecaw** [comment]
Hmm looks like a duplicated request that allready exists here
https://github.com/Zrips/CMI/issues/3870

I dont think this portable beacon by command is a good idea. But the others sounds locical no matter if its used or not
